src/Servers/Kestrel/shared/KnownHeaders.cs

```diff
@@ -110,7 +100,8 @@ static KnownHeaders()
                 HeaderNames.CorrelationContext,
                 HeaderNames.TraceParent,
                 HeaderNames.TraceState,
-                HeaderNames.Baggage
+                HeaderNames.Baggage,
+                HeaderNames.Pragma,
```

Comment:
Why was Pragma moved? We use it as a response header:
https://github.com/dotnet/aspnetcore/blob/c4d06ecd2aa45ea246b66038f314c23b48b8db4b/src/Security/Authentication/Cookies/src/CookieAuthenticationHandler.cs#L402
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Pragma

Reply:
Bump @benaadams

Reply:
Want to revisit post https://github.com/dotnet/aspnetcore/pull/31519 as a merge conflict in this PR would be easy to resolved; but painful in the other PR

Reply:
Moved it back